test/src/main/java/org/apache/accumulo/test/metrics/fate/ZooLockTest.java

```diff
@@ -57,16 +57,19 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-public class ZooLockIT extends SharedMiniClusterBase {
+public class ZooLockTest {
+
+  private static ZooKeeperTestingServer szk = null;
 
   @BeforeClass
   public static void setup() throws Exception {
-    SharedMiniClusterBase.startMiniCluster();
+    szk = new ZooKeeperTestingServer();
+    szk.initPaths("/accumulo/1234");
```

Comment:
You may want to consider using UUID.randomUUID().toString() - and if the UUID is used elsewhere, create / initialize it as a static final variable for the class.  This more closely emulates the Accumulo instance id / paths used by Accumulo. 

With a quick glance I think because you are using the `super.create(path, data, acl, createMode);` later - will also create a path that is `/accumulo/[instance-id]`

Reply:
Good idea. Will include this in my next commit.